i18n/sample/src/Profiles.i18n.json

```diff
@@ -6,9 +6,11 @@
     "createNewConnection.option.prompt.url": "Enter a z/OSMF URL in the format 'https://url:port'.",
     "createNewConnection.enterzosmfURL": "No valid value for z/OSMF URL. Operation Cancelled",
     "createNewConnection.option.prompt.userName.placeholder": "Optional: User Name",
-    "createNewConnection.option.prompt.userName": "Enter the user name for the connection",
-    "createNewConnection.option.prompt.password.placeholder": "Optional: Password",
-    "createNewConnection.option.prompt.password": "Enter a password for the connection",
+    "createNewConnection.option.prompt.userName": "Enter the user name for the connection. Leave blank to not store.",
+    "createNewConnection.undefined.enteruserName": "Operation Cancelled",
+    "createNewConnection.option.prompt.passWord.placeholder": "Optional: Password",
+    "createNewConnection.option.prompt.password": "Enter the password for the connection. Leave blank to not store.",
+    "createNewConnection.undefined.enterpassWord": "Operation Cancelled",
```

Comment:
should these properties be camelCased like `enterUsername` `enterPassword`
And also, on the placeholder, should password be all lower case? (`prompt.password.placeholder`)

Reply:
fixed 👍